app/components/works/edit/files_component.html.erb

```diff
@@ -0,0 +1,13 @@
+<table class="table table-sm" id="filesTable">
+    <thead class="table-light">
+      <tr>
+        <th width="50%">Filename</th>
+        <th>Label</th>
+        <th width="10%">Hide <%= render PopoverComponent.new key: 'work.hide_file' %></th>
```

Comment:
I think this may need to be "Description" to match what is in the item show page (and the designs for that show page): https://projects.invisionapp.com/share/YQ132HXFX2GR#/screens/471005875